jwst/datamodels/schemas/msa_targ_acq.schema.yaml

```diff
@@ -0,0 +1,200 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/msa_targ_acq.schema"
+title: MSA target acquisition data model
+allOf:
+- $ref: core.schema
```

Comment:
Probably don't need to inherit `core.schema` here, because at some point this schema will itself be inherited into a higher-level schema that most likely already includes core.

Reply:
Removing that inherit statement.

Reply:
The reference to core.schema is still in there and needs to be removed.